armi/physics/thermalHydraulics/settings.py

```diff
@@ -17,10 +17,19 @@
 from armi.settings import setting
 from armi.operators.settingsValidation import Query
 
+CONF_DO_TH = "doTH"
+
 
 def defineSettings():
     """Define generic thermal/hydraulic settings."""
-    settings = []
+    settings = [
+        setting.Setting(
+            CONF_DO_TH,
+            default=False,
+            label="Run Thermal Hydraulics",
+            description="Run thermal hydraulics",
```

Comment:
🦆 Might need a better description. In the global ARMI scope, what does this really mean in a context where any plugin might respond to it? Should we jump straight to a `thKernel` setting instead?